packages/e2e-contract-tests/playwright/e2e/DepositHalfEth.test.ts

```diff
@@ -41,43 +39,47 @@ test.describe('Deposit Half ETH', () => {
     await page.waitForTimeout(3000);
     await depositHalfButton.click();
 
-    const walletPage = await getWalletPage(context);
+    const walletNotificationPage =
+      await fuelWalletTestHelper.getWalletNotificationPage();
```

Comment:
would be better `getWalletPopupPage`?

```suggestion
    const walletPopupPage =
      await fuelWalletTestHelper.getWalletPopupPage();
```